apps/judicial-system/web/src/routes/Prosecutor/Indictments/Defendant/PoliceCaseInfo/PoliceCaseInfo.tsx

```diff
@@ -123,6 +126,29 @@ export const PoliceCaseInfo: FC<Props> = ({
     updatePoliceCase,
   ])
 
+  useEffect(() => {
+    if (
+      index !== indexDate.index &&
+      crimeScene?.date?.getTime() !== indexDate.date?.getTime()
+    ) {
+      // Our position in the list has changed because we changed the crime scene date
+      const dateElement = document.getElementById(crimeSceneDateId)
```

Comment:
A more "react-y" way to do this is to define a `ref` in this component, pass it down to `DateTime` as prop and set the `DateTime` ref there. Then you'd be able to use that ref to scroll into view and you'd also be able to use [a hook](https://github.com/streamich/react-use/blob/master/docs/useIntersection.md) to determine if that element is in view. Please consider making these changes. I could also try to see if it works if you are preoccupied in something else :)